### ISSUE TITLE
CSS-in-JS nested media query position interpolation: no-duplicate-selectors false positive

### ISSUE THREAD

**Issue by klarstrup** [stylelint/stylelint#6115]
### What steps are needed to reproduce the bug?

```js
const small = "@media (min-width: 400px)";

const OfferPcs = styled.div`
  ${small} {
    display: none;
  }
  > div {
    ${small} {
      display: none;
    }
  }
`;
```

<img width="741" alt="image" src="https://user-images.githubusercontent.com/1567633/170247580-ab2ffe9c-7611-4201-a3a1-8b752069916e.png">

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "@stylelint/postcss-css-in-js",
  "extends": ["stylelint-config-recommended", "stylelint-config-styled-components"]
}
```

### How did you run Stylelint?

`stylelint .`

### Which version of Stylelint are you using?

`14.8.4`

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problem was reported:
```
Offer.tsx
  33:5  ✖  Unexpected duplicate selector "", first used at line 25  no-duplicate-selectors
```

### Does the bug relate to non-standard syntax?

Yes, it's related to CSS-in-JS & nested properties

**Comment by klarstrup** [comment]
This is fixed as of 14.8.5 which was published as I was writing this up lmao
